src/components/chart/boxplot.js

```diff
@@ -0,0 +1,66 @@
+import ChartConstants from './chart-constants';
+
+class Boxplot {
+    constructor() {
+        this.constants = ChartConstants.constants();
+    }
+
+    getBoxplotOptions = () => {
+        return {
+            plotOptions: {
```

Comment:
Are we missing setting the options for the column spacing here? It should match the column chart.